packages/terra-disclosure-manager/src/DisclosureManager.jsx

```diff
@@ -444,7 +461,7 @@ class DisclosureManager extends React.Component {
     }
 
     return render({
-      dismissPresentedComponent: (disclosureComponentKeys.length > 1) ? this.generatePopFunction(disclosureComponentKeys[disclosureComponentKeys.length - 1]) : this.safelyCloseDisclosure,
+      dismissPresentedComponent: (disclosureComponentKeys.length > 0) ? this.generatePopFunction(disclosureComponentKeys[disclosureComponentKeys.length - 1]) : () => {},
```

Comment:
This is a little weird, but I'm forcing this function to go through the pop logic rather than deferring to `safelyCloseDisclosure` when we're popping the last item. I'm providing a no-op function in the alternate flow for passivity, but I'm fairly certain this API isn't being used by anyone so 🤷‍♂️

Reply:
This is a little odd. We can always assume we can use 
```
this.generatePopFunction(disclosureComponentKeys[disclosureComponentKeys.length - 1]
```
here with the checks for the key values to match?

Either way we should add an inline comment to describe why this is being done

Reply:
If we use generatePopFunction all time, we'll end up calling it with `undefined` when there are no disclosure keys present. It might be worthwhile to just throw an `undefined` check along with the key equality and use it all the time.

Reply:
It seems cleaner to me to just use generatePopFunction all the time with the `undefined` check.

Reply:
I agree with @jmsv6d on just moving this check to the `generatePopFunction`. Then we could (not sure if needed) update the comment for `generatePopFunction`

Reply:
Updatd: b66b17e